slices/finalrd.yaml

```diff
@@ -0,0 +1,33 @@
+package: finalrd
+
+essential:
+  - finalrd_copyright
+
+slices:
+  # finalrd is a shell script that relies on a couple of packages to even
+  # work correctly: libc-bin, systemd, mount, grep and coreutils.
+  # If initramfs-tools is supplied (which is optional), it also needs
+  # findutils and debianutils. so rely on atleast the first set of binaries.
+  bins:
+    essential:
+      - coreutils_bins
```

Comment:
Since this is a shell script should we add `dash_bins` for the shell and `base-files_bin` for `/bin`?

Reply:
you probably don't need the entire `coreutils`, so I suggest creating a new coreutils slice(s) with just the bins you need for this.